### ISSUE TITLE
The resulting image is wrong

### ISSUE THREAD

**Issue by zhujuntong** [lichen2018/stVAE#2]
I would like to ask the author why the final output of my mannul file is a gray image without any dots. I hope to get your reply!

**Comment by lichen2018** [comment]
Did you use the mannul.ipynb file?

**Comment by zhujuntong** [comment]
Yes, I ran the file and the final image was a grey plot without any points with numerical values

**Comment by lichen2018** [comment]
Have you downloaded the file st_mouse_brain_20.h5ad in the command "st_adata_raw=anndata.read_h5ad('/project/Stat/s152403/stVAE/test/st_mouse_brain_20.h5ad')" from google driver link https://drive.google.com/drive/folders/1OspGGEcPNBoqG8_8MYZC_gXPbt034wIl?

**Comment by zhujuntong** [comment]
Thank you very much for your reply! I later went to check and found that the format of the data assignment in the code did not match, which eventually led to the situation of no data. Now the problem has been solved, thank you very much for your guidance!
